jupyter-notebooks/analytics/user-guide/03_change_detection.ipynb

```diff
@@ -44,12 +44,12 @@
    "source": [
     "## 2. Change Detection Results\n",
     "\n",
-    "The semantic change detection product has been built through the following steps:\n",
+    "Planet's Road & Building Change Detection product has been built through the following steps:\n",
     "\n",
     "1. Develop a high performing road and building detection model and leverage it to extract roads and buildings from imagery\n",
     "2. Run road and building detection over an AOI for an extended period of time, gathering as many data points as possible. From month to month, variations in atmosphere, shadows and other factors cause very small differences in the outputs over time leading to significant noise or “flicker”\n",
-    "3. We de-noise the results by considering the temporal context of each pixel detected. This leads to a more consistent output, however it does introduce a “lag” in when we are able to detect change\n",
-    "4. We then compare the most recent de-noised output with the previous de-noised output, pull out the differences, \"decimate\" the changes to a fixed size, and polygonize them.\n",
+    "3. We de-noise the results by considering the temporal context of each pixel detected. This leads to a more consistent output, however it does introduce a \"lag\" in when we are able to detect change\n",
+    "4. We then compare the most recent de-noised output with the previous de-noised output, pull out the differences and vectorize them into polygons, or more specifically into grid cells. This has the benefit of signaling where our model has detected a change.\n",
```

Comment:
Are these cells on a standardized grid? Is it web mercator?

Reply:
No; it's a computer vision decimation algorithm, where adjacent cells are 'decimated' and upscaled to be larger blocks. I'm not quite sure exactly what the decimation kernel size is -- we probably don't want to print it here in case it changes in the future. The decimation means that the decimated 'grid' cells can be arbitrary placed based on where the original pixel based change detections happened.